docs/source/index.rst

```diff
@@ -1,8 +1,8 @@
 .. _rubicon:
 
-=======
-Rubicon
-=======
+===========
+rubicon_ml
```

Comment:
so now we have a PyPi project called `rubicon-ml` and our library, `rubicon_ml`. I'd say that this is where we wanna use the dash name `rubicon-ml` since its the project's name and this is the docs' title. but I can totally see how changing between the two could get confusing. thoughts?

Reply:
haha this is what i was debating too -- i went with the library name since it's the docs for the library but i could see how it can go either way.. maybe we should flip a coin 😂